lib/search.js

```diff
@@ -98,6 +98,11 @@ function filter (data, args, notArgs) {
 }
 
 function stripData (data) {
+  // Sanitize the maintainers field to ensure that NPM repositories (Artifactory) that
+  // does not sanitize this won't break the search functionality.
+  if (typeof data.maintainers === 'string') {
+    data.maintainers = [data.maintainers]
+  }
   return {
     name: data.name,
     description: npm.config.get('description') ? data.description : '',
```

Comment:
ah but either way that breaks down at usage, since `m.name` isn't gonna work if its a string.